datetime/src/test/java/com/fasterxml/jackson/datatype/jsr310/deser/LocalDateTimeDeserTest.java

```diff
@@ -60,14 +60,23 @@
     private final TypeReference<Map<String, LocalDateTime>> MAP_TYPE_REF = new TypeReference<Map<String, LocalDateTime>>() { };
 
     final static class StrictWrapper {
-        @JsonFormat(pattern="yyyy-MM-dd HH:mm",
+        @JsonFormat(pattern="yyyy-MM-dd HH:mm G",
```

Comment:
Same comment as previously, this format should stay 'as-is' (without the era), so we still test the original test case (which is the common scenario), and then have new test cases for the 'year-with-era' scenario ('yyyy-MM-dd HH:mm G') and the 'year' scenario ('uuuu-MM-dd HH:mm')

Reply:
Same as the above, the pattern `yyyy-MM-dd HH:mm` doesn't allow any valid date-time(like `2019-11-30 15:45`) in strict mode, would you mind me update the original test case to say `test the invalid format` rather `test the invalid date time`. And cover the `invalid date time` in other test cases.

Reply:
Yes, sounds good.